libnavui-androidauto/build.gradle

```diff
@@ -41,8 +41,8 @@ dependencies {
     // This defines the minimum version of Maps, Navigation, and Search which are included in
     // this SDK. To upgrade the SDK versions, you can specify a newer version in your downstream
     // build.gradle.
-    def carNavVersion = "2.7.0-rc.2"
-    def carSearchVersion = "1.0.0-beta.34"
+    def carNavVersion = "2.8.0-alpha.3"
```

Comment:
@Zayankovsky is there anything else you want to try to add to `2.7`?

We're attempting to get a stable api that will be at least 2.8. We don't have multiple release channels so we will essentially stop targeting 2.7 after this.

Reply:
Oh i thought 1tap was still on 2.7, but looked it up and it's on 2.8 too. So good to go here